### DISCUSSION TITLE
No contracts for requested resources

### DISCUSSION THREAD

**Discussion by BastianWel** [International-Data-Spaces-Association/DataspaceConnector#116] [Q&A] [answered]
I have requested a resource (`POST
 /api/ids/description` & `POST
 /api/ids/contract`) that has a n-times-usage contract.
Then I see this ressource with `GET /api/requests`, but `GET
 /api/requests/{id}/contracts` returns an empty contracts-array.
Shouldn't I see the contracts of the requested resource there?

**Comment by juliapampus** [comment, marked as answer]
No, this is intended beviour as there is no need that a data consumer stores the contract offers of a data provider. Both have a contract agreement that is the only relevant contract information.

**Reply by BastianWel** [reply]
Thank you, then I will use the contract information from the agreement.
